core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_body_directive.html

```diff
@@ -131,4 +140,36 @@
     text-align: center;
     text-shadow: -1px 0 white, 0 1px white, 1px 0 white, 0 -1px white;
   }
+
+  simple-editor-body .question-type-wrapper {
+    background-color: white;
```

Comment:
I think @kevinlee12 left a comment to the contrary, but actually we are trying to represent all colors as hex or rgba numbers. Could you please change them back? Sorry about that.